src/cr/cube/matrix.py

```diff
@@ -1424,7 +1529,25 @@ def values(self):
 
     @lazyproperty
     def zscore(self):
-        return self._zscore
+        variance = (
```

Comment:
These also look exactly the same as the above calculations... But I'm not sure if they can be moved up the hierarchical structure. If they can, please move them. If not, this is fine too.